test/test_custom_ops.py

```diff
@@ -361,39 +363,34 @@ def test_invalid_schemas(self):
         # function schmea validation goes through torchgen, so this is just a
         # basic test.
         with self.assertRaisesRegex(AssertionError, "Invalid function schema: foo"):
-
-            @custom_op(f"{TestCustomOp.test_ns}::foo", "(")
-            def foo(x):
-                ...
+            custom_ops.custom_op(f"{TestCustomOp.test_ns}::foo", "(")
 
     def test_name_must_match(self):
         with self.assertRaisesRegex(ValueError, "to have name"):
 
-            @custom_op(f"{TestCustomOp.test_ns}::foo", "(Tensor x) -> Tensor")
-            def bar(x):
-                ...
-
-        with self.assertRaisesRegex(ValueError, "to have name"):
-
-            @custom_op(f"{TestCustomOp.test_ns}::foo")
+            @custom_ops.custom_op(f"{TestCustomOp.test_ns}::foo")
             def baz(x: Tensor) -> Tensor:
-                ...
+                raise NotImplementedError()
```

Comment:
The alternative for this is that we ask the user to write code that looks like the following
```
def baz(x: Tensor) -> Tensor:
    ...  # noqa: F111
```
(or whatever the number is). There is no way for us (pytorch, the framework) to disable the mypy lint rule in user code.

Suggesting the user to ignore a mypy lint rule doesn't sound like a good user experience.

Reply:
Ehh I agree that NotImplementedError() looks weird though, because you'd expect someone to fill in the blanks. Maybe it should be a different type of error?

Reply:
Hmmm maybe? I don't have any better ideas though at the moment.